lib/webserver/WebServer.js

```diff
@@ -447,10 +447,10 @@ const WebServer = function (options) {
     });
 
     /* New function with support of current zone cleanup */
-    this.app.put("/api/start_cleaning_zone_by_name", async (req, res) => {
+    this.app.put("/api/start_cleaning_zones_by_id", async (req, res) => {
```

Comment:
why you changed the api from names to ids?
I think its better to stay with names.

Reply:
Both Roborock and Viomi APIs operate on IDs. The UI operates on IDs. If users were to give different zones the same name (or the robot-provided name matches a user-defined name) we run into problems. I think it’s much safer to go with IDs.